tests/basic/storage_pool_test.go

```diff
@@ -8588,6 +8588,23 @@ var _ = Describe("{AddDiskAddDriveAndDeleteInstance}", func() {
 	})
 })
 
+// Checks if journal device present on the node
+func isJournalPresentOnNode(n *node.Node) (bool, error) {
```

Comment:
please use GetJournalDevicePath function for getting journal device path on a node

Reply:
done